Gemfile

```diff
@@ -53,6 +55,14 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+gem 'jquery-rails'
+gem 'record_tag_helper', '~> 1.0'
+gem 'devise'
+gem 'bootstrap-generators', '~> 3.3.4'
+gem 'friendly_id', '~> 5.2.1'
+gem 'commontator', '~> 5.1.0'
```

Comment:
Neat, I hadn't seen this gem before! Does it support nested comments at all?

Reply:
Hi !
Hope you're doing good;
It doesnt support nested comments as of now; If it was a requiremnent I would have looked for something else or probably implemented it using something like `ancestry`